### PR TITLE
Fix SVG problems in Meshery-UI

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
Reverts meshery/meshery#6637

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
This PR solves SVG scaling problem in safari and remove the temporary solution of icon library.
1. Introduce consistency by reverting [this PR](https://github.com/meshery/meshery/pull/6637).
2. When an icon is used directly from the libraries like MUI or font-awesome or something else, always specify the height and width prop.
3. Repeat these changes on the existing and newer svg components

**Review by leecalcote — CHANGES_REQUESTED** [review]
@Abhishek-kumar09 given that you're changing icons, please screenshot these changes in context of the pages in which they are used. Share screenshots in the PR, please.

**Comment by Abhishek-kumar09** [comment]
There are some set of SVG styles [specifying common height and width](https://github.com/meshery/meshery/pull/6690/files#diff-06f61349be806f176b8c47957b6dc8f701835c0ba8ef8313106488a361833aa8)

All the icons imported from any library, could be either small, or medium, or large, or xl. One of these styles based on icon demand should be applied to all those svgs.

Examples of how to do it: 
- https://github.com/meshery/meshery/pull/6690/commits/ecc260461b6015ca680d6969e01bfa9565f373f2
- https://github.com/meshery/meshery/pull/6690/commits/4b000e07a2eeb70c402085cdf86544783bcc939c

Files Done:

1. ui/components/CatalogFilter.js
2. ui/components/ConfirmationModal.js
3. ui/components/Content.js
4. ui/components/CustomTableFooter.js
5. ui/components/DashboardComponent.js
6. ui/components/ErrorHandling.js
7. ui/components/Filters.js
8. ui/components/Header.js
9. ui/components/MeshAdapterConfigComponent.js
10. ui/components/MesherySettings.js
11. ui/components/MeshConfigComponent.js
12. ui/components/MesheryAdapterPlayComponent.js
13. ui/components/MesheryApplications.js
14. ui/components/MesheryMetrics.js
15. ui/components/MesheryNotification.js

**Comment by Abhishek-kumar09** [comment]
Screenshot 2023-01-16 at 8 49 31 PM
Screenshot 2023-01-16 at 8 50 14 PM
Screenshot 2023-01-16 at 8 50 17 PM
Screenshot 2023-01-16 at 8 50 28 PM
Screenshot 2023-01-16 at 8 50 32 PM
Screenshot 2023-01-16 at 8 50 41 PM
Screenshot 2023-01-16 at 8 50 52 PM
Screenshot 2023-01-16 at 8 51 08 PM
Screenshot 2023-01-16 at 8 49 57 PM

File: ui/css/icons.styles.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// icon styles, setting general height and width properties to solves scaling and consistency problems
```
Comment (Chadha93) [inline ui/css/icons.styles.js:1]: @Abhishek-kumar09 Did you also test this on browser zoom in and out on Safari? Are the icons scaling? With the defined dimensions?

Comment (Abhishek-kumar09) [reply ui/css/icons.styles.js:1]: Yes, I did it.

**Review by Chadha93 — APPROVED** [review]
LGTM

**Comment by Abhishek-kumar09** [comment]
> LGTM, some failing tests to look.....

This is coming from the next-upgrade

**Pull request merged**